README.md

```diff
@@ -51,7 +50,7 @@
 
 Visit the site live [here.](https://mysite-28qd.onrender.com)
 
-You can see the video presentation for this project [here.](https://www.loom.com/share/8f12e2777cdc4a198036debc24bcd1)
+You can see the video presentation for this project [here.](https://www.loom.com/share/95ffe9e384d94fb2a06becb5a1f6ab67?sid=e2230fb9-3077-4cd7-beca-f0f4e06864bb)
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you highlight a piece of the code that you are proud of making by showing us a little walkthrough of that part of your code. You mentioned it in the video but did not show us. Let's fix this